reviews/2022-metallb-annual.md

```diff
@@ -0,0 +1,102 @@
+
+MetalLB works with both relatively simple networking topology using the L2 mode that does not require any additional network component, or using the BGP protocol when BGP enabled routers are available.
+
+More information about the MetalLB architecture can be found [here](https://metallb.universe.tf/concepts/)
+
+## Development metrics
+
+The MetalLB devstats page and dashboards can be found [here](https://metallb.devstats.cncf.io/).
+
+Here are some links worth mentioning:
+
+- [Stars and forks](https://metallb.devstats.cncf.io/d/3/stars-and-forks-by-repository?orgId=1&from=now-1y&to=now)
+- [Issues](https://metallb.devstats.cncf.io/d/12/issues-opened-closed-by-repository-group?orgId=1)
+- [PRs](https://metallb.devstats.cncf.io/d/15/new-prs-in-repository-groups?orgId=1&var-period=d&var-repogroup_name=All)
+- [Commits](https://metallb.devstats.cncf.io/d/2/commits-repository-groups?orgId=1)
+
+## Maintainers
+
+There are three maintainers actively working on the project (alphabetically):
+
+| Maintainer          | GitHub ID                                         | Affiliation |
+| ------------------- | ------------------------------------------------- | ----------- |
+| Federico Paolinelli | [fedepaol](https://github.com/fedepaol)           | Red Hat     |
+| Ori Braunshtein     | [oribon](https://github.com/oribon)               | Red Hat     |
+| Graeme Lawes        | [gclawes](https://github.com/gclawes)             | Nasdaq      |
+
+## Adoption
+
+MetalLB is probably the most popular solution for exposing services of type LoadBalancer in bare metal deployments. It's being distributed with Red Hat's Openshift
+since [version 4.9](https://docs.openshift.com/container-platform/4.9/networking/metallb/about-metallb.html), and widely used by users running kubernetes in homelabs
+(an [issue opened](https://github.com/metallb/metallb/issues/1481#issuecomment-1176716132) agains MetalLB revealed a wide community of users in the k8s@home discord channel).
+MetalLB is part of [multiple tutorials and videos](https://www.google.com/search?q=metallb+kubernetes+tutorial) written by the community and suggested by multiple projects
+such as [kubespray](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/metallb.md), [kind](https://kind.sigs.k8s.io/docs/user/loadbalancer/) and [microk8s](https://microk8s.io/docs/addon-metallb) (to mention some).
+
+Despite not having a clear estimate of the number users / companies consuming MetalLB, the images related to the [controller](https://quay.io/repository/metallb/controller) / [speaker](https://quay.io/repository/metallb/speaker) components show an average of
+40k "actions" per day for the controller image (which is deployed in single instance) and 150k "actions" per day for the speaker image.
+
```

Comment:
You could also reference https://github.com/metallb/metallb/issues/5 for some reports the project has received. You could also reference any product docs you know about where metallb is included ([Red Hat](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html/networking/load-balancing-with-metallb), [Microsoft](https://learn.microsoft.com/en-us/azure-stack/aks-hci/deploy-metallb), maybe some others?).

Reply:
Turns out there are others, I wasn't aware of that.